_posts/2021-04-18-api-conversion.md

```diff
@@ -0,0 +1,85 @@
+The main reason why Strimzi users had to do this migration are preparations of Strimzi to use [Kubernetes CRD `apiextensions/v1`](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122).
+
+Custom resources are created and controlled using APIs added to Kubernetes by CRDs.
+Put another way, CRDs extend the Kubernetes API to allow the creation of custom resources.
+CRDs are themselves resources within Kubernetes.
+They are installed in a Kubernetes cluster to define the versions of API for the custom resource.
+Each version of the custom resource API can define its own schema for that version.
+Kubernetes clients, including the Strimzi Operators, access the custom resources served by the Kubernetes API server using a URL path (API path), which includes the API version
+
+In 0.22, we still support custom resource version `v1alpha1` for all custom resources and `v1beta1` for `Kafka`, `KafkaConnect`, `KafkaConnectS2I`, `KafkaMirrorMaker`, `KafkaUser` and `KafkaTopic`.
+However, these versions are currently deprecated and will be dropped in the following Strimzi 0.23.
+That's one of the reasons, why users have to finish the migration before 0.23.
+
+One another change is, that with new API version we remove deprecated fields so users need to migrate them to new format.
+For instance `v1beta2` supports only new array based listener configuration and new metrics configuration formats from config maps.
+`metrics` and `logging` field were also changed as well.
+For metrics configuration you should use `metricsConfig` in `spec.kafka` to reference name of config map with metrics configuration in `valueFrom` field.
+Same for `logging`, but field name remains same.
+In addition, some old unused fields like `tlsSidecar` were removed as well.
+All changes in fields are described in our [documentation](https://strimzi.io/docs/operators/latest/full/deploying.html#proc-upgrade-cli-tool-files-str).
+You can also see the deprecated field in CR status as warnings.
+
+For smooth migration to new CRs and CRDs version we created `api-conversion` tool. _Note that for proper conversion you need to have installed Strimzi 0.22 with all its CRDs._
+
+### API conversion tool
+
+The tool is shipped as part of Strimzi 0.22 in [zip](https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.22.1/api-conversion-0.22.1.zip) and [tar.gz](https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.22.1/api-conversion-0.22.1.tar.gz) archives.
+After un-archiving, you will find there all libs needed by the tool in `libs` directory and two scripts for running the tool in folder `bin`.
+Script `api-conversion.sh` is for linux/mac users and `api-conversion.cmd` is for Windows users.
+
+The tool can operate in two modes:
+* convert your resources to `v1beta2`. It can convert resources in yaml files or resource already deployed into Kubernetes cluster.
```

Comment:
```suggestion
* Converting custom resources to `v1beta2`. It can convert resources in yaml files or resource already deployed into Kubernetes cluster.
```